### DISCUSSION TITLE
Ideas for Live Stats window

### DISCUSSION THREAD

**Discussion by JJWatMyself** [f1multiviewer/issue-tracker#174] [Feature requests] [answered]
Auto hide play-pause controls until mouse over
Add  Play-pause buttons to play-pause all streams
Clickable Button to sync all streams
Pop-out track map
Keyboard shortcuts shift space to play-pause all streams
Keyboard shortcut s to sync all streams
Keyboard shortcut r to toggle hide-show race control messages

**Comment by jeppeolesen** [comment, marked as answer]
Most of these have been implemented. Pease see the docs for keyboard shortcuts: https://multiviewer.dev/docs
